vendor/flarum/tags/js/src/forum/addTagLabels.js

```diff
@@ -12,6 +12,13 @@ export default function() {
     const tags = this.props.discussion.tags();
 
     if (tags && tags.length) {
+      if(tags.length==1){
+        items.add('tags',{...tagsLabel(tags, {link:false}, 10), itemClassName: "item-tags-single"})
+      }
+      else if(tags.length==2){
+        items.add('tags',{...tagsLabel(tags, {link:false}, 10), itemClassName: "item-tags-double"})
+      }
+      else
       items.add('tags', tagsLabel(tags), 10);
```

Comment:
Chỗ này em bị duplicate code. Em có thể chỉnh bằng cách tạo 1 biến để giữ cái  itemClassName` thì như vậy em không phải gọi `items.add` ở mỗi cái `if`. Anh khuyên nên làm như bên dưới:

```suggestion
      let itemClassName = ""
      if (tags.length == 1) {
        itemClassName = "item-tags-single"
      }
      else if (tags.length == 2) {
         itemClassName = "item-tags-double"
      }
      items.add('tags', { ...tagsLabel(tags), itemClassName}, 10);
```